app/components/repo-actions.js

```diff
@@ -62,6 +63,7 @@ export default Component.extend({
   canCancel: and('userHasPullPermissionForRepo', 'item.canCancel'),
   canRestart: and('userHasPullPermissionForRepo', 'item.canRestart'),
   canDebug: and('userHasPushPermissionForRepo', 'item.canDebug'),
+  isQueued: reads('item.notStarted'),
```

Comment:
As far as I can see from the template code, there's no need to define an alias here, it's short enough to just use `item.notStarted` directly in template. What do you think?

Reply:
Done.